gnovm/pkg/gnolang/preprocess.go

```diff
@@ -9,6 +9,8 @@ import (
 	"github.com/gnolang/gno/tm2/pkg/errors"
 )
 
+const blankIdentifer string = "_"
```

Comment:
I agree on adding this, but can you do `s/"_"/blankIdentifier/g`?

Reply:
Will add this in a subsequent PR